app/services/book_service.go

```diff
@@ -0,0 +1,26 @@
+package services
+
+import (
+	"github.com/ilie20088/go-web-app-boilerplate/app/models"
+	"github.com/ilie20088/go-web-app-boilerplate/app/repositories"
+)
+
+// BookFetcher is an interfaces that defines set of business operations related to books
+type BookFetcher interface {
+	FetchBook(id string) (*models.Book, error)
+}
+
+// BookFetcherImpl is an implementation of BookFetcher
+type BookFetcherImpl struct{}
+
+var bookRepository repositories.BookRepository
+
+// InitBookService initializes book service
+func InitBookService(_bookRepository repositories.BookRepository) {
+	bookRepository = _bookRepository
+}
+
+// FetchBook fetches book by id or returns a error returned by book repository
+func (b BookFetcherImpl) FetchBook(id string) (*models.Book, error) {
+	return bookRepository.FetchBookByID(id)
+}
```

Comment:
@momoneko Performance. Found this http://stackoverflow.com/questions/27775376/value-receiver-vs-pointer-receiver-in-golang . At the moment, we don't have any fields in struct. So its zero-memory allocation. But this might change in future.